pkg/authorization/cache/clusterpolicy.go

```diff
@@ -24,7 +24,7 @@ type readOnlyClusterPolicyCache struct {
 	keyFunc cache.KeyFunc
 }
 
-func NewReadOnlyClusterPolicyCache(registry clusterpolicyregistry.WatchingRegistry) readOnlyClusterPolicyCache {
+func NewReadOnlyClusterPolicyCache(registry clusterpolicyregistry.WatchingRegistry) *readOnlyClusterPolicyCache {
```

Comment:
> ah, I think that's the bug... cache.NewReflector() returns *Reflector... we should preserve that as a pointer

Alright.  Pointers for everyone!  @stevekuznetsov do that in addition to this change.